docs/python/generators/cpp_parser.py

```
@@ -133,6 +138,13 @@ def __init__(self, name: str, doc: str | None, definition: str, location: str):
     def namespaced_name(self):
         return self.name
 
+# Fingerprint of parser data-model classes so cache self-invalidates on code changes.
+_CACHE_FINGERPRINT = hashlib.md5(
```

Ruff (0.15.21)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)